src/property_validator.coffee

```diff
@@ -2,7 +2,7 @@
 # ------------------
 
 module.exports = class PropertyValidator
-  termRegex = /\w[\w ]*\w/g
+  termRegex = /[^, ][^,]*[^, ]/g
```

Comment:
hehe. that works for everything...
except for validator names that consist of a single char ;)